src/l1-resources/search-index/README.md

```diff
@@ -42,22 +42,42 @@ You can find more information about activating this type in the [AWS CloudFormat
 import { CfnSearchIndex } from 'awscdk-resources-mongodbatlas';
 
 const mySearchIndex = new CfnSearchIndex(this, 'MySearchIndex', {
-    analyzer: 'lucene.standard',
-    clusterName: atlasProps.clusterName,
-    collectionName: atlasProps.collectionName,
-    database: atlasProps.dbName,
-    mappings: {
-        fields: [
-        "summary:string", 
-        "description:string",
-        "minimum_nights:number"],
-        dynamic: false,
-    },
-    name: atlasProps.indexName,
-    profile: atlasProps.profile,
-    projectId: atlasProps.projId,
-    searchAnalyzer: 'lucene.standard'
-    });
+  profile: atlasProps.profile,
+  projectId: atlasProps.projId,
+  clusterName: atlasProps.clusterName,
+  name: atlasProps.indexNameSearch,
+  collectionName: atlasProps.collectionName,
+  database: atlasProps.dbName,
+  searchAnalyzer: 'lucene.standard',
+  analyzer: 'lucene.standard',
+  mappings: {
+      fields: JSON.stringify({
+        employees: {
+          type: "string",
+          analyzer: "lucene.whitespace",
+        }
+      }),
+      dynamic: false,
+  },
+});
+
+const myVectorSearchIndex = new CfnSearchIndex(this, 'MyVectorSearchIndex', {
+  profile: atlasProps.profile,
+  projectId: atlasProps.projId,
+  clusterName: atlasProps.clusterName,
+  name: atlasProps.indexNameVector,
+  collectionName: atlasProps.collectionName,
+  database: atlasProps.dbName,
+  type: 'vectorSearch',
+      fields: JSON.stringify([
+        {
+          type: "vector",
+          path: "plot_embedding",
+          numDimensions: 1536,
+          similarity: "euclidean"
+        }
+      ]),
+});
 ```
 
 ## Feedback
```

Comment:
in the below: * Issues related to this generated library should be [reported here](https://github.com/cdklabs/cdk-cloudformation/issues/new?title=Issue+with+%40cdk-cloudformation%2Fmongodb-atlas-searchindex+v1.0.0) should be updated to v3.0.0

Reply:
thx, updated